test/integration/deployment/util.go

```diff
@@ -206,40 +208,55 @@ func markPodReady(c clientset.Interface, ns string, pod *v1.Pod) error {
 	return err
 }
 
-// markUpdatedPodsReady manually marks updated Deployment pods status to ready
-func (d *deploymentTester) markUpdatedPodsReady() {
+func intOrStrP(num int) *intstr.IntOrString {
+	intstr := intstr.FromInt(num)
+	return &intstr
+}
+
+// markUpdatedPodsReady manually marks updated Deployment pods status to ready,
+// until the deployment is complete
+func (d *deploymentTester) markUpdatedPodsReady(wg *sync.WaitGroup) {
+	defer wg.Done()
```

Comment:
When building PRs based on other PRs, it's recommended to pick up the commits you need into your PR, instead of directly copying codes. It's easier to rebase and keeps those commits separate.

Reply:
I am aware of it, but it is too late to pick up your change. AFAIK, I came out with the PR before yours, and I expected your PR to merge before mine.